.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,33 @@
-Please include the following information in your ticket.
+Please [read these guidelines](http://ibm.biz/cdt-issue-guide) before opening an issue.
 
-- Cloudant (python-cloudant) version(s) that are affected by this issue.
-- Python version 
-- A small code sample that demonstrates the issue.
+<!-- Issues will be CLOSED IMMEDIATELY if the following template is not completed. -->
+
+## Bug Description
+
+### 1. Steps to reproduce and a small code sample that demonstrates the issue
+<!--
+EXAMPLE:
+1. Initialize client
+2. List DBs
+
+cc = new CloudantClient(...)
+dbNames = cc.allDbs()
+-->
+
+### 2. What you expected to happen
+<!--
+Expected to get a list of all databases
+-->
+
+### 3. What actually happened
+<!--
+Got a 400 bad request
+-->
+
+## Environment details
+<!--
+- Version(s) that are affected by this issue.
+    > 2.0.2
+- Python version
+    >3.6.2
```

Comment:
Add space between `>` and `3.6.2`.

Reply:
Fixed in eba45bb591cad33951571d3d7067e0021817dbca